pageserver/src/layered_repository/inmemory_layer.rs

```diff
@@ -227,6 +227,32 @@ impl Layer for InMemoryLayer {
     fn is_incremental(&self) -> bool {
         self.img_layer.is_some()
     }
+
+    /// debugging function to print out the contents of the layer
+    fn dump(&self) -> Result<()> {
```

Comment:
How can this function be used? It is not accessible from `dump_layerfile` since InMemoryLayer doesn't have files.
Is it supposed to be called manually while debugging?

Reply:
Wasn't sure how to address this @hlinnaka

Reply:
Right, it's dead code currently. It can be used manually while debugging. Or we can replace it with a dummy implementation that does nothing.